docs/content/docs/connectors/table/pulsar.md

```diff
@@ -413,5 +415,61 @@ CREATE TABLE users
     'key.json.ignore-parse-errors' = 'true',
     'key.fields' = 'uid;item_id',
     'value.format' = 'json',
-    'value.json.fail-on-missing-field' = 'false',
+    'value.json.fail-on-missing-field' = 'false'
 ```
+
+### Configure auth parameters
+
+This example shows how to specify and configure the auth parameters used by the underlying Pulsar admin client. T
+
+```sql
+CREATE TABLE users
+(
+  `uid`         BIGINT,
+  `item_id`     BIGINT,
+  `description` STRING
+) WITH (
+    'connector' = 'pulsar',
+    'topics' = 'persistent://public/default/users',
+    'service-url' = 'pulsar://localhost:6650',
+    'admin-url' = 'http://localhost:8080',
+    'pulsar.client.tlsAllowInsecureConnection' = 'true', 
+    'pulsar.client.authPluginClassName' = 'org.apache.pulsar.client.impl.auth.AuthenticationToken',
+    'pulsar.client.authParams' = 'token:eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1c2VyMSJ9.2AgtxHe8-2QBV529B5DrRtpuqP6RJjrk21Mhn'
+```
+
+
+## Relationship between apache/flink and streamnative/flink
+
+StreamNative maintains a fork of [apache/flink](https://github.com/apache/flink) in [streamnative/flink](https://github.com/streamnative/flink). The StreamNative's fork only maintains a different Pulsar DataStream Connector and Pulsar SQL Connector compared with the official Flink repository.
+
+streamnative/flink contains both Pulsar DataStream Connector and Pulsar SQL Connector. However, apache/flink contains only Pulsar DataStream Connector at this time. We will merge back the Pulsar SQL Connector in the coming months. For Pulsar DataStream Connector, the code and distribution in streamnative/flink differs slightly from those in apache/flink. Usually, new features and bug fixes will first go into streamnative/flink, and then PR is created against apache/flink.
+
+For detailed differences and recommendations, please see the following sections.
+
+### Components
+
+| Repository              | Pulsar DataStream Source | Pulsar DataStream Sink | Pulsar SQL Source | Pulsar SQL Sink |
+|-------------------------|--------------------------|------------------------|-------------------|-----------------|
+| apache/flink 1.14       | Yes                      |                        |                   |                 |
+| apache/flink 1.15       | Yes                      | Yes                    |                   |                 |
+| streamnative/flink 1.15 | Yes                      | Yes                    | Yes               | Yes             |
```

Comment:
We may also add Flink 1.14 support with the Pulsar 2.10.0.

Reply:
Can the 1.15.0 and above connector be used with Flink 1.14 directly? I haven't tested yet.